### ISSUE TITLE
feature : search bar 

### ISSUE THREAD

**Issue by EslemOuederni** [EslemOuederni/transportation-app-dashboard#13]
**The problem**
The dashboard has many tables that will be filled with lots of data. So It'll be hard to look for specific data through many rows.r us.

**The solution**
 In order to search for a specific person's name in the tickets table, or search for trips we need to implement a search bar to make it easier for us.

**Additional context**
the search bar would be implemented in the header in certain pages such as tickets Page,  trips Page, user's page

Any help would be appreciated ! 
Thank you !

**Comment by Srinja333** [comment]
can  i work on this?

**Comment by EslemOuederni** [comment]
yes sure @Srinja333 , I'll assign you to the issue. If you face any problem, please let me know.

**Comment by Srinja333** [comment]
@EslemOuederni  ok

**Comment by EslemOuederni** [comment]
@Srinja333 thank you for updating me ! Good luck with your exams

**Comment by NIKHILNAIR21** [comment]
can i work on this?

**Comment by EslemOuederni** [comment]
@NIKHILNAIR21 hello there, yes of course. Thank you for the suggestion.

**Comment by EslemOuederni** [comment]
Hi @NIKHILNAIR21, are you still working on this issue ? just so that I know if I still assign you to it or not

**Comment by NIKHILNAIR21** [comment]
Yes

**Comment by EslemOuederni** [comment]
Thank you !!!

**Comment by NIKHILNAIR21** [comment]
unable start server with 
npm run dev

npm ERR! Missing script: "dev"
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\nikhi\AppData\Local\npm-cache\_logs\2023-03-24T04_45_57_964Z-debug-0.log

**Comment by EslemOuederni** [comment]
Hello, did you follow up the installation guide ? 
`cd client ->npm install && npm run dev`
`cd server -> npm install && npm start`

**Comment by NIKHILNAIR21** [comment]
where will i get the mongodb url to add in .env file and their is no  config file in server folder

**Comment by EslemOuederni** [comment]
You can make your own mongodb database for the testing purposes. There, you ll get a URL that you can use.

**Comment by NIKHILNAIR21** [comment]
ok

**Comment by singhkaran202** [comment]
Hey @EslemOuederni , Can I work on this issue, if that's OK?

**Comment by EslemOuederni** [comment]
@singhkaran202, yes sure !
